apps/website/src/pages/docs/variables.mdx

```diff
@@ -215,8 +277,25 @@ The opacity scale decreases from 1 to 6, with `--iui-opacity-1` being fully opaq
 
 <OpacityDemo />
 
-### Shadows
+### Shadow
 
-iTwinUI offers a few shadows that increase in elevation from 0 to 5. These should typically be used in combination with the `Surface` component.
+iTwinUI offers a few shadows that increase in elevation from 0 to 5. These variables are typically used in combination with the [surface component](surface).
 
 <ShadowsDemo />
+
+### Duration
+
+If you need to animate anything, use one of the supplied duration variables. Please use these variables in combination with the [`prefers-reduced-motion` media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) for accessibility reasons. For most hover or click interactions we encourage using the fastest speed.
+
+<DurationDemo />
+
+## Resources
+
+<RelatedComponents
```

Comment:
Not needed as it's mentioned in the first paragraph.

Reply:
Removed.